contrib/inventory/gce_googleapiclient.py

```diff
@@ -24,51 +25,59 @@
 
 Before using:
 
-- Authentication: this script uses the same authentication as gcloud command line. So, set it up before.
-- Dependencies: it depends mainly on google-api-python-client and docoptcfg. So:
+- Authentication: this script uses the same authentication as
+                  gcloud command line. So, set it up before.
+- Dependencies: it depends mainly on google-api-python-client
+                and docoptcfg. So:
 
 $ pip install google-api-python-client docoptcfg
 
 Usage:
-    {script_name} [options]
+    {script_name} (--project=PROJECT...|--all-projects --all-zones --billing-account=ACCOUNT_NAME)
```

Comment:
that's good to be able to use multiple projects. but `--all-zones` aren't required when using `--all-projects`. you can always request all projects on a specific zone or all zones on a specific project.